app/models/job_invocation.rb

```diff
@@ -15,7 +15,8 @@ class JobInvocation < ActiveRecord::Base
   validates :job_name, :presence => true
   validates_associated :targeting, :template_invocations
 
-  scoped_search :on => :job_name
+  scoped_search :on => :job_name, :complete_value => true
+  scoped_search :on => :description, :complete_value => true
```

Comment:
If you look at @adamruzicka's example, default descriptions render quotes around the input values.  I think it makes sense to remove the auto complete for now.

Filed https://github.com/wvanbergen/scoped_search/issues/138